docs/contracts/v3/reference/deployments/Celo-Deployments.md

```diff
@@ -0,0 +1,60 @@
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------- |
+| [UniswapV3Factory](https://github.com/Uniswap/uniswap-v3-core/blob/v1.0.0/contracts/UniswapV3Factory.sol)                                                    | `0xAfE208a311B21f13EF87E33A90049fC17A7acDEc` |
+| [Multicall2](https://celoscan.io/address/0x633987602de5c4f337e3dbf265303a1080324204#code)                                                                    | `0x633987602DE5C4F337e3DbF265303A1080324204` |
+| [ProxyAdmin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/ProxyAdmin.sol)                                   | `0xc1b262Dd7643D4B7cA9e51631bBd900a564BF49A` |
+| [TickLens](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/lens/TickLens.sol)                                                          | `0x5f115D9113F88e0a0Db1b5033D90D4a9690AcD3D` |
+| [NFTDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/libraries/NFTDescriptor.sol)                                           | `0xa9Fd765d85938D278cb0b108DbE4BF7186831186` |
+| [NonfungibleTokenPositionDescriptor](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungibleTokenPositionDescriptor.sol)           | `0x644023b316bB65175C347DE903B60a756F6dd554` |
+| [TransparentUpgradeableProxy](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.1-solc-0.7-2/contracts/proxy/TransparentUpgradeableProxy.sol) | `0x505B43c452AA4443e0a6B84bb37771494633Fde9` |
+| [NonfungiblePositionManager](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/NonfungiblePositionManager.sol)                           | `0x3d79EdAaBC0EaB6F08ED885C05Fc0B014290D95A` |
+| [V3Migrator](https://github.com/Uniswap/uniswap-v3-periphery/blob/v1.0.0/contracts/V3Migrator.sol)                                                           | `0x3cFd4d48EDfDCC53D3f173F596f621064614C582` |
+| [QuoterV2](https://github.com/Uniswap/v3-periphery/blob/main/contracts/lens/QuoterV2.sol)                                                                    | `0x82825d0554fA07f7FC52Ab63c961F330fdEFa8E8` |
+| [SwapRouter02](https://github.com/Uniswap/swap-router-contracts/blob/main/contracts/SwapRouter02.sol)                                                        | `0x5615CDAb10dc425a742d643d949a7F474C01abc4` |
+| [Permit2](https://github.com/Uniswap/permit2)                                                                                                                | `0x000000000022D473030F116dDEE9F6B43aC78BA3` |
+| [UniversalRouter](https://github.com/Uniswap/universal-router)                                                                                               | `0x643770E279d5D0733F21d6DC03A8efbABf3255B4` |
```

Comment:
also not verified... but celo i remember usually didnt work for verification 😆

Reply:
yeah, I was thinking of either making a push to get some of these non-verified ones verified (just-toby also brought up a lot), or making a script and doing it myself, but yeah another one that gave them issues when launching. [verified in SDK here](https://github.com/Uniswap/universal-router/blob/main/deploy-addresses/celo.json)